english/33hours/wakingup.md

```diff
@@ -1,10 +1,11 @@
 The sun on your face is the first thing you become aware of and you instinctively put your hand up to shield your eyes
-from the brightness. You're still in your clothes and are lying in the middle of a bed in what looks to be some sort of
-cheap motel room. You have no idea where you are or how you got here.
+from the brightness. You're still in your
+clothes. And you are lying in the middle of a bed in what looks to be some sort [of](../Harold/Harold.md) cheap motel room. You have no
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.